lib/from-layout.ts

```diff
@@ -349,6 +349,13 @@ function ProvableFromLayout<
       check(value: T): void {
         check(typeData, value);
       },
+      // TODO implement properly
```

Comment:
Thanks for adding a reminder in the comments. Do you mind also creating an issue for this so it doesn't get lost?